docs/ref/contrib/contenttypes.txt

```diff
@@ -543,3 +543,25 @@ information.
 
     Subclasses of :class:`GenericInlineModelAdmin` with stacked and tabular
     layouts, respectively.
+
+.. module:: django.contrib.contenttypes.management
+
+Utilities
+=========
+.. versionadded:: 1.9
+
+The `remove_contenttypes` method can be used to programmatically remove stale content types. This is useful for automatic deployments where keyboard input isn't available. 
+
+It accepts one argument, the application name, and is called within the migration file.
+
+.. code-block:: python
+
+    # 0002_my_migration.py
+    from django.contrib.contenttypes.management import remove_contenttypes
+
+    def remover(apps, schema_editor):
+        remove_contenttypes('my_app')
+
+    operations = [
+        migrations.RunPython(remover),
+        ]
```

Comment:
```
+    operations = [
+        migrations.RunPython(remover),
+    ]
```

instead of

```
+    operations = [
+        migrations.RunPython(remover),
+        ]
```